src/test/regress/citus_tests/config.py

```diff
@@ -230,9 +230,6 @@ def __init__(self, arguments):
             "local_dist_join_load",
             "local_dist_join",
             "arbitrary_configs_recurring_outer_join",
-            # group 7
-            "sequences_create",
-            "sequences",
```

Comment:
as we've discussed earlier, let's not remove this since it still fails due to limited planner support